main/main.c

```diff
@@ -47,6 +47,10 @@ void app_main(void)
     } else {
         ESP_LOGE(TAG, "Invalid DEVICE model");
         // maybe should return here to now execute anything with a faulty device parameter !
+        // this stops crashes/reboots and allows dev testing without an asic
+        GLOBAL_STATE.device_model = DEVICE_UNKNOWN;
+        GLOBAL_STATE.asic_count = 1;
```

Comment:
why not putting 0 default value here, instead of assuming there is a actual chip in a actual voltage domain, better shutting everything down by using 0 default value

Reply:
My bad, it should have been -1 as a canary, 0 is fine with me too.